docs/docs/50-user-guide/60-reference-docs/70-annotations.md

```diff
@@ -23,6 +23,7 @@ programmatically with Kargo.
 | `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | A string that is unique or at least unlikely to repeat, such as a UUID or a timestamp of "now" | Triggers reconciliation of the resource when its value changes. |
 | `kargo.akuity.io/reverify` | `Stage` | Either a plain string (verification ID from `.status.verifications[*].id` of the `Stage`) or a JSON object with `id` (required), `actor`, and `controlPlane` fields. If a JSON object is provided, it is parsed as a `VerificationRequest`. | Triggers re-verification of a previously completed verification for the current `Freight`. |
 | `rbac.kargo.akuity.io/claim.<name>` | `ServiceAccount` | Any valid OIDC claim value (e.g., `sub`, `email`, or `groups`) | Maps an OIDC claim to a `ServiceAccount`, enabling user-to-ServiceAccount mappings. For more details, refer to the access control sections of the [Operator Guide](../../40-operator-guide/40-security/30-access-controls.md) and [User Guide](../50-security/20-access-controls/index.md). |
```

Comment:
I want to stop short of deprecating this, but I think it's worth mentioning that the new `rbac.kargo.akuity.io/claims` annotation is the recommended way of doing this, although I don't think you need to get into why that is.